backend/Exceptions/File/FileException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Exceptions\File;
+
+use Internships\Exceptions\SimpleMessageException;
+use Throwable;
+
+class FileException extends SimpleMessageException
+{
+    protected string $fullPath;
+    protected string $fileName;
```

Comment:
Where is this variable used? 
`protected string $fileName;`

Reply:
Oh, it's little leftover. Before I wanted to have path and filename separate, then I noticed that filename itself can be included with path.
Already removed in my local branch. I'll push with status code change.